pallets/manta-pay/src/lib.rs

```diff
@@ -487,42 +506,57 @@ pub mod pallet {
         fn pull_receivers_for_shard(
             shard_index: u8,
             receiver_index: usize,
+            max_update: u64,
             receivers: &mut ReceiverChunk,
+            receiver_pulled: &mut u64,
```

Comment:
```suggestion
            receivers_pulled_count: &mut u64,
```

Reply:
The fact that it is a count  is implied in the type. Will change to plural form tho.

Reply:
disagreeing on the technicality that numeric type doesn't imply count, especially not in the singular variable name form - when i read this I thought it meant a receiver index.
i favor the explicit naming, but simple plural works too